tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -265,8 +265,14 @@ All elements of a position section are serialized as Ints
 
 Standard Section: "Comments" Comment*
 ```none
-  Comment       = Length Bytes LongInt      // Raw comment's bytes encoded as UTF-8, followed by the comment's coordinates.
+  Comment       = UTF8 LongInt              // Raw comment's bytes encoded as UTF-8, followed by the comment's coordinates.
 ```
+
+Standard Section: "Attributes" Attribute*
+```none
+  Attribute     = FLAGattr UTF8             // attributes match the regex [a-zA-Z0-9]+
```

Comment:
I might not have been clear enough here about what I meant by specifying what attributes we can have. I suggest we keep an actual list of the possible UTF-8 strings we can find here, and is their meaning. So something like:
```suggestion
  Attribute     = FLAGattr UTF8             // attributes match the regex [a-zA-Z0-9]+
  
  Possible values of FLAGattr attributes are:
  * Scala2StandardLibrary    This TASTy contains code from the Scala 2 standard library
```
Then later on we can also add:
```none
  * ExplicitNulls      This TASTy uses the explicit-nulls variant of the type system
```

Reply:
if we do this then why store utf-8 at all, and go fully to tags

Reply:
I will use the tags only as we currently only need them for core Scala features. If we need we can add a tag that can store arbitrary UTF-8 later.